test/performance/serializing_test.rb

```diff
@@ -0,0 +1,16 @@
+require "benchmark_helper"
+
+class SerializingTest < ActionDispatch::PerformanceTest
+  # Refer to the documentation for all available options
+  # self.profile_options = { runs: 5, metrics: [:wall_time, :memory],
+  #                          output: 'tmp/performance', formats: [:flat] }
+
+  test "checkins_page" do
+    dev = Developer.first
```

Comment:
The other way could be using factories for user/developer here. But in this case we'll have to include DB cleaner stuff also.